org.flycast.Flycast.yml

```diff
@@ -0,0 +1,36 @@
+app-id: org.flycast.Flycast
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: flycast
+rename-desktop-file: flycast.desktop
+rename-icon: flycast
+finish-args:
+  - --device=all
+  - --filesystem=host:ro
```

Comment:
why "host" ? this is a very broad permssion.

Reply:
Like other emulators out there, Flycast needs to read wherever the location of the ROM/BIOS files could be.